lib/package.json

```diff
@@ -81,17 +81,17 @@
     "cross-env": "^5.1.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.0",
-    "eslint": "^4.8.0",
+    "eslint": "^4.17.0",
     "eslint-config-airbnb": "^16.0.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.5.1",
     "fs-extra": "^4.0.2",
     "glob": "^7.1.2",
     "jest": "^21.2.1",
-    "material-ui": "^1.0.0-beta.31",
+    "material-ui": "^1.0.0-beta.33",
```

Comment:
looks like you forgot to upgrade material-ui version in peerDependencies (should be beta.32)